Client/Frontend/Settings/FxAContentViewController.swift

```diff
@@ -55,10 +55,10 @@ class FxAContentViewController: SettingsContentViewController, WKScriptMessageHa
 
     deinit {
         NotificationCenter.default.removeObserver(self, name: NotificationFirefoxAccountVerified, object: nil)
-        
-        if AppConstants.MOZ_SHOW_FXA_AVATAR {
-            profile.getAccount()?.updateProfile()
-        }
+//        
+//        if AppConstants.MOZ_SHOW_FXA_AVATAR {
+//            profile.getAccount()?.updateProfile()
```

Comment:
Why is this commented out?

Reply:
[Commit comment][1]:  

>  I'm not sure why this happens in a deinitializer, but it is causing a…
> … crash. (it's being executed on the wrong thread).

This definitely needs to be resolved before this lands.


[1]: 481698ffb1aded144239e5a88ed322feb801a552

Reply:
Just to give a little background, this was added so that the avatar/display name would get updated after the user navigates from FxA webview settings page (they could have changed it). Where might be a better place to put this?